src/Task/Base/ServiceProvider.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Robo\Task\Base;
+
+use League\Container\ServiceProvider\AbstractServiceProvider;
+
+class ServiceProvider extends AbstractServiceProvider
+{
+    protected $provides = [
```

Comment:
$provides is required by league/container's service provider.  The 'register' function isn't called until someone calls 'get' on a provided service.